apps/src/sites/studio/pages/components/FlexGroup.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+import color from '../../../../util/color';
+import { borderRadius } from './constants';
+import OrderControls from './OrderControls';
+import StageCard from './StageCard';
+
+const styles = {
+  groupHeader: {
+    fontSize: 18,
+    color: 'white',
+    background: color.cyan,
+    borderTopLeftRadius: borderRadius,
+    borderTopRightRadius: borderRadius,
+    padding: 10
+  },
+  groupBody: {
+    background: color.lightest_cyan,
+    borderBottomLeftRadius: borderRadius,
+    borderBottomRightRadius: borderRadius,
+    padding: 10,
+    marginBottom: 20
+  },
+  addGroup: {
+    fontSize: 14,
+    color: 'white',
+    background: color.cyan,
+    border: `1px solid ${color.cyan}`,
+    boxShadow: 'none',
+    margin: '0 0 30px 0'
+  },
+  addStage: {
+    fontSize: 14,
+    color: '#5b6770',
+    background: 'white',
+    border: '1px solid #ccc',
+    boxShadow: 'none',
+    margin: '0 10px 10px 10px'
+  }
+};
+
+const FlexGroup = React.createClass({
+  propTypes: {
+    addGroup: React.PropTypes.func.isRequired,
+    addStage: React.PropTypes.func.isRequired,
+    stages: React.PropTypes.array.isRequired
+  },
+
+  handleAddGroup() {
+    this.props.addGroup();
+  },
```

Comment:
I think you could just do `onMouseDown={this.props.addGroup}` instead of creating a wrapper method.